store4s/src/main/scala/store4s/Datastore.scala

```diff
@@ -23,21 +23,26 @@ case class Datastore(underlying: GDatastore) {
   def get(key: Key) = Option(
     underlying.get(key, Seq.empty[ReadOption]: _*)
   )
-  def get(key: Seq[Key]) = Option(
-    underlying.get(key.asJava, Seq.empty[ReadOption]: _*)
-  )
+  def get(key: Seq[Key]) = underlying.get(key.asJava, Seq.empty[ReadOption]: _*)
   def getEither[A: WeakTypeTag: EntityDecoder](name: String) = {
     get(keyFactory[A].newKey(name)).map(decodeEntity[A])
   }
   def getEither[A: WeakTypeTag: EntityDecoder](id: Long) = {
     get(keyFactory[A].newKey(id)).map(decodeEntity[A])
   }
+  def getEithers[A: WeakTypeTag: EntityDecoder](nameList: Seq[String]) = {
```

Comment:
To support both name and id, let's change it to `getEithersByName` and `getEithersById` then.